src/main/resources/static/api.yaml

```diff
@@ -855,3 +872,34 @@ components:
             attribute: true
           format: <user-id>@<ncg-id>
           example: batman@ncg
+
+    LinkedCareContexts:
+      type: object
+      properties:
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/links'
+    links:
+      type: object
+      properties:
+        hip:
+          type: object
+          properties:
+            id:
+              type: string
+            name:
+              type: string
+        careContexts:
+          type: array
+          items:
+            $ref: '#/components/schemas/MaskedCareContexts'
+    MaskedCareContexts:
+      type: object
+      properties:
+        referenceNumber:
+          type: string
+        hint:
```

Comment:
Is hints relevant anymore. this shows already linked HIPs and CCs.  Hint is relevant only during linking process. Are we keeping Hint Text in CM db post linkages?